### PR TITLE
Create IVR with Autopilot 

### PR DESCRIPTION
**Pull request by maylonpedroso**
- frontend setup page
- assistant setup saved in a JSON file using `lowdb`
- asynchronous update request to Twilio APIs

### CODE REVIEW

File: routes/autopilot.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+/* eslint-disable new-cap */
+const router = express.Router();
+
+// POST: /collect/main-menu
+router.post('/collect-main-menu', function(req, res, next) {
+  const options = JSON.parse(req.body.Memory).twilio.collected_data.options;
+  const answer = options.answers.option.answer;
+  if (options.status === 'complete') {
+    if (['sales', 'support'].includes(answer)) {
```
Comment (mrcrow85) [inline routes/autopilot.js]: I think a `switch` will be better than an `if..else`:
```javascript
switch(answer) {
  case 'sales':
  case 'support':
    ...
    break;
  case 'operator':
    ...
    break;
  default:
    res.send({
      ...
    });
}
```

File: package.json [hunk 2]
```diff
@@ -32,24 +33,26 @@
...
-    "cookie-parser": "~1.4.3",
-    "debug": ">=2.6.9",
-    "dotenv-safe": "^4.0.4",
+    "cookie-parser": "~1.4.4",
+    "debug": ">=4.1.1",
+    "dotenv-safe": "^8.2.0",
     "express": "^4.17.1",
+    "lowdb": "^1.0.0",
+    "moment": "^2.24.0",
     "morgan": "^1.9.1",
```
[the remark this answers is not in the document]
Comment (maylonpedroso) [reply package.json]: `morgan` is in use: https://github.com/TwilioDevEd/sample-autopilot-voice-ivr/blob/master/server.js#L5

[the remark this answers is not in the document]
Comment (mrcrow85) [reply package.json]: Ah yeah, sorry. My bad, didn't see that

File: routes/autopilot.js (hunk 1 above)
Comment (maylonpedroso) [reply routes/autopilot.js]: :100:

File: src/assistant.js [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+
+/**
+ * Create or update the Main Menu task for the assistant
+ * @param {AssistantInstance} assistant
+ * @param {TaskInstance[]} tasks
+ * @param {string} baseUrl
+ * @return {Promise<TaskInstance>}
+ */
+async function createOrUpdateMainMenuTask(assistant, tasks, baseUrl) {
+  let task = tasks.find(task => task.uniqueName === 'main-menu');
```
Comment (mrcrow85) [inline src/assistant.js]: Same here, as with `assistant`, we should avoid mutating objects as much as possible.

File: src/assistant.js [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+    actions.actions.push({
+      listen: { tasks: nextTasks },
+    });
+  } else {
+    actions.actions.push({
+      redirect: `task://${nextTasks[0]}`,
+    });
+  }
+
+  let task = tasks.find(task => task.uniqueName === uniqueName);
```
Comment (mrcrow85) [inline src/assistant.js]: Same here with `task`, lets use `const` instead

**Pull request merged**